src/test_lib_json/main.cpp

```diff
@@ -3373,7 +3373,7 @@ struct CharReaderAllowDropNullTest : JsonTest::TestCase {
   Value emptyArray = Value{Json::arrayValue};
 
   ValueCheck checkEq(const Value& v) {
-    return [=](const Value& root) { JSONTEST_ASSERT_EQUAL(root, v); };
+    return [=, this](const Value& root) { JSONTEST_ASSERT_EQUAL(root, v); };
```

Comment:
Is `this` really used by the lambda? It looks like it should only need to capture `v`.

Reply:
It looks like the expansion of JSONTEST_ASSERT_EQUAL uses "JsonTest::TestCase::result_"